public/views/content/entryForm.html

```diff
@@ -4,7 +4,7 @@
     <div class="col-sm-6">
       <div class="form-group">
         <label for="date">Date:</label>
-        <input type="date" id="date" class="form-control" ng-model="newEntry.date" value="{{ date | date: 'yyyy-MM-dd' }}" />
+        <input data-provide="datepicker" type="text" class="form-control datepicker" ng-model="newEntry.date" value="{{ date | date: 'mm/dd/yyyy' }}">
```

Comment:
So I just checked mlab and the format for saving dates get changed. We were initiaally processing the date object server side and changing it to UNIX millis. It is not doing that anymore. Is the format in which HTML form is sending it the same?